Microsoft.Azure.CosmosRepository/src/InMemoryRepository.cs

```diff
@@ -198,6 +199,12 @@ public async ValueTask<bool> ExistsAsync(Expression<Func<TItem, bool>> predicate
                 item => item.Type == typeof(TItem).Name, Expression.AndAlso).Compile());
         }
 
+        public ValueTask<IPage<TItem>> ScrollAsync(int lastPage = 0, Expression<Func<TItem, bool>> predicate = null, int pageSize = 25, string continuationToken = null,
+            CancellationToken cancellationToken = default)
```

Comment:
```suggestion
        public ValueTask<IPage<TItem>> ScrollAsync(
            int page = 0,
            Expression<Func<TItem, bool>> predicate = null,
            int pageSize = 25,
            string continuationToken = null,
            CancellationToken cancellationToken = default)
```

Format nit, and parameter name suggestion.